torch/xpu/streams.py

```diff
@@ -96,6 +96,29 @@ def __repr__(self):
         return f"torch.xpu.Stream(device={self.device} sycl_queue={self.sycl_queue:#x})"
 
 
+class ExternalStream(Stream):
+    r"""Wrapper around an externally created SYCL queue.
+
+    This class is used to wrap SYCL queue created in other libraries in order
+    to facilitate data exchange and multi-library interactions.
+
+    .. note:: This class doesn't manage the queue life-cycle, it is the user
+       responsibility to keep the referenced queue alive while this class is
+       being used. The different SYCL queue pointers will result in distinct
+       ExternalStream objects, even if they reference the same underlying SYCL
+       queue object.
+
+    Args:
+        queue_ptr(int): Integer representation of the `sycl::queue*` value passed externally.
```

Comment:
Should we highlight here again. The queue_ptr to a SYCL queue and its copies will generate distinct ExternalStream objects.

Reply:
I have highlighted it in ..note. which could be rendered. I think it is enough.